src/dx/presence/builders/heartbeat.rs

```diff
@@ -339,4 +344,45 @@ mod it_should {
 
         assert!(matches!(result, Err(PubNubError::EffectCanceled)));
     }
+
+    // TODO: Make request cancelable
```

Comment:
Do we want to release this with this PR?

Reply:
Probably next one will be fine :)